packages/platforms/accton/arm64/as4224/src/modules-5_6/arm64-accton-as4224-psu.c

```diff
@@ -45,6 +45,14 @@
 static ssize_t show_count(struct device *dev, struct device_attribute *da, char *buf);
 static ssize_t show_status(struct device *dev, struct device_attribute *da, char *buf);
 extern int as4224_cpld_read(unsigned short cpld_addr, u8 reg);
+extern int as4224_cpld_write(unsigned short cpld_addr, u8 reg, u8 value);
+
+enum as4224_platform_id {
+    AS5114_48X,
```

Comment:
Is there a better solution to have *as4224* in the enum name, but *AS5114_48X* as a member?

Reply:
The enum is renamed as enum Platform_Id.